web/conftest.py

```diff
@@ -818,9 +818,7 @@ def _assert_num_queries(db="default", **expected_counts):
                         stack_frames = (
                             q["stack"]
                             if pytestconfig.getoption("verbose") > 1
-                            else [q["userland_stack_frame"]]
-                            if q["userland_stack_frame"]
-                            else []
+                            else q.get("userland_stack_frame", [])
```

Comment:
This fixes a bug with this query assertion; passing the `-v` flag is meant to show all queries that ran, but this line could never succeed if `userland_stack_frame` was missing from the dict.